src/hera.cpp

```diff
@@ -92,7 +92,10 @@ struct hera_instance : evmc_instance {
   bool metering = false;
   map<evmc_address, vector<uint8_t>> contract_preload_list;
 
-  hera_instance() noexcept : evmc_instance({EVMC_ABI_VERSION, "hera", hera_get_buildinfo()->project_version, nullptr, nullptr, nullptr, nullptr, nullptr}) {}
+  // hera_instance() noexcept : evmc_instance({EVMC_ABI_VERSION, "hera", hera_get_buildinfo()->project_version, nullptr, nullptr, nullptr, nullptr, nullptr}) {}
+  // fix error with geth+evmc:  /tmp/go-build940146972/b001/runtime.test: symbol lookup error: /tmp/go-build940146972/b001/runtime.test: undefined symbol: hera_get_buildinfo
+  hera_instance() noexcept : evmc_instance({EVMC_ABI_VERSION, "hera", nullptr, nullptr, nullptr, nullptr, nullptr, nullptr}) {}
```

Comment:
@chfast it seems like `buildinfo` doesn't work in static linking?!

Reply:
Hopefully, addressed by https://github.com/ewasm/hera/pull/517.